### PR TITLE
adding discuss kubernetes link to community page

### PR DESCRIPTION
**Pull request by jimangel**
addressing [#8578](https://github.com/kubernetes/website/issues/8578)

### CODE REVIEW

**Comment by jimangel** [comment]
I can update the link text from "Discussion Board" to "Forum" to better inline with castrojo: https://github.com/kubernetes/website/pull/8945

Thoughts?

File: content/en/community/_index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <main>
         <div class="content">
             <h3>Ensuring Kubernetes works well everywhere and for everyone.</h3>
-            <p>Connect with the Kubernetes community on our <a href="http://slack.k8s.io/">Slack channel</a> or join the
+            <p>Connect with the Kubernetes community on our <a href="http://slack.k8s.io/">Slack channel</a>, <a href="https://discuss.kubernetes.io/">Discussion Board</a>, or join the
```
Comment (mistyhacks) [inline content/en/community/_index.html]: Maybe format these as a bullet list and talk about what each one is best for? Also I don't think "discussion board" needs to be capitalized.

**Comment by mistyhacks** [comment]
/lgtm
/approve

It would be good in future to expand this to show what communication mechanism is best for what purpose.

**Pull request merged**